projects/core/src/lib/spatial-references/spatial-reference.service.ts

```diff
@@ -154,14 +155,15 @@ export class SpatialReferenceService {
             map((dict: SpatialReferenceSpecificationDict) => {
                 const spec = SpatialReferenceSpecification.fromDict(dict);
 
-                proj4.defs(spec.projString);
+                proj4.defs(spec.spatialReference.srsString, spec.projString);
```

Comment:
the units are shown correctly now, but I noticed that before your change, the background layer for UTM32N e.g. would be just black, and after your change it is now a distorted background layer that does not make much sense to me.

Do you have an idea why this is? Before and after your change the console says "Using fallback background layer for EPSG:32632".